wesnoth-nr.po

```diff
@@ -3154,15 +3041,13 @@ msgstr ""
 msgid ""
 "Warning! These bottles contain toxic waste. Causes disintegration on contact."
 msgstr ""
-"Pozor! Tyto lahvičky obsahují toxický odpad, který způsobuje při doteku "
-"rozpadání."
+"Pozor! Tyto lahvičky obsahují toxické slitky, které způsobují při kontaktu "
```

Comment:
Lahvička obsahuje slitek? Na to mi fantazie nestačí. Proč tam nenechat odpad?

Reply:
No mě tam celkově ten toxický odpad nesedí. Proč by se jako odpad (např. z alchymistických, kouzelnických či jiných experimentů) měl dávat (dávkovat) do lahviček? To by mohlo být jedině tehdy, pokud by lahvička sloužila na slévání odpadu. Pak ale obsahuje slitky :-).
Kdyby tam bylo, že obsahují toxické odpadní slitky, bylo by to lepší? Nebo jde o to, že slovo slitky znáš jen ve významu pevných (kovových) produktů slévání?

Reply:
Teď koukám, že slitky ve formě kapaliny jsou nějaký žargon mého pracovního okolí. Omlouvám se. Daly by se použít slivky?

Reply:
Odpad v lahvích nebo lahvičkách jsem nedávno odvážel na sběrný dvůr. Tedy nebezpečné látky v lahvích si dokážu představit.

Ano, je to proto, že si představuji kovové slitky.

Na kapalné slitky si můžu zvyknout, ten význam po vysvětlení chápu, ale nevím, co jiní (text hry by neměl nutit k přemýšlení o významu jednotlivých slov).

Za slivkami vidím jen švestky (ještě víc teď po sklonění, a co teprve v jednotném čísle).

Reply:
Nakonec:
Tyto lahvičky obsahují kapalný toxický odpad, který způsobuje při kontaktu poleptání.

Mohlo by se zdát, že to kapalný je navíc. Ale asi se shodneme, že v lahvičce je z hledika kontextu kapalina a ne třeba pasta nebo plyn. Takže jsem si dovolil tam nechat něco, co zdůrazňuje kapalnost (a také zůstává mírná reminiscence na ty slitky).